bootloaders/riotboot/Makefile

```diff
@@ -8,7 +8,7 @@ BOARD ?= samr21-xpro
 FEATURES_REQUIRED += riotboot
 
 # Disable unused modules
```

Comment:
This comment looks somewhat misplaced now

Reply:
Good point. I have changed this.